docs/_content/doorontwikkeling/index.md

```diff
@@ -4,6 +4,19 @@ date: '15-7-2019'
 weight: 10
 ---
 
+## Verwacht
+
+De publicatie van de 1.1.0 versies voor alle voor Zaakgericht Werken in gebruik zijnde API standaarden wordt nog in 2019 verwacht. Daarmee zullen een aantal belangrijke features aan deze API standaarden worden toegevoegd. Denk daarbij aan caching voor al deze API standaarden en ondersteuning voor bestanden groter dan 3 Gb in de Documenten API standaard. Indien gewenst kunnen deze vernieuwingen nu al bestudeerd worden in de testomgevingen van deze API standaarden:
```

Comment:
> De publicatie van de 1.1.0 versies voor alle voor Zaakgericht Werken in gebruik zijnde API standaarden wordt nog in 2019 verwacht.

Ik heb 3 "problemen" met deze zin:

1. Het is niet waar dat er 1.1.0 versies komen voor ALLE ZGW API's. De Notificaties API heeft 0 wijzigingen.
2. Het impliceert (hoewel deels terecht maar dat terzijde) dat de de API's niet los van elkaar gezien kunnen worden. De term ZGW moet komen te vervallen m.i.
3. De vraag is nog maar of we dit in 2019 gaan redden. Release van API's behalve Klantinteractie staat niet op de sprint backlog en kost ook tijd.

Ik zou het enigszins herschrijven naar:

> Na de release van 1.0.0 van de eerste VNG API's worden de API's vanaf nu apart doorontwikkeld. Dit zal tot gevolg hebben dat van sommige API's op relatief korte termijn een 1.1.0 versie zal verschijnen waarin een [aantal belangrijke features ...]

Reply:
Ik heb je wijzigingen overgenomen.